plugins/gestures.koplugin/main.lua

```diff
@@ -41,6 +41,8 @@ local gestures_list = {
     hold_top_right_corner = _("Top right"),
     hold_bottom_left_corner = _("Bottom left"),
     hold_bottom_right_corner = _("Bottom right"),
+    one_finger_swipe_down = _("Down"),
```

Comment:
Possibly a candidate for a context (something like `"Swipe"`).

Reply:
I also thought that just one word would look a bit odd. But then the enclosing menu item `One-finger swipe` makes any extra words redundant.

Reply:
Pardon, I meant `C_()`, not within the phrase itself.

http://koreader.rocks/doc/modules/gettext.html#pgettext

Reply:
I see, another possibility to avoid the ambiguity altogether is to use an icon as is done for the two-finger swipes.

Reply:
The ambiguity is for translation, not for users.

Reply:
![Screenshot_2021-04-20-21-51-40-50_1aff61e6bd981aaa4ba9d0245eb6c04f](https://user-images.githubusercontent.com/202757/115455769-ad98eb80-a222-11eb-93cd-35c37e7e1161.jpg)

Reply:
that's definitely better since the diagonal one-finger swipes could also be added in the future, right?